src/react-apps/applications/ux-editor/src/types/global.ts

```diff
@@ -83,6 +83,7 @@ declare global {
     required?: boolean;
     hidden?: boolean;
     readOnly?: boolean;
+    [key: string]: string | any;
```

Comment:
What does this structure represent? Maybe create an interface for that specific structure with a name, making a bit more readable?

Reply:
This is set in order to write "component[value] " in CollapsibleMenuComponent to toggle an unknown param of 'IFormComponent'.
This error will show: Element implicitly has an 'any' type because type 'IFormComponent' has no index signature.